app/views/layouts/_emails.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if user.additional_email.blank? %>
+  <%= mail_to user.email, user.email %>
+<% else %>
+  <%= mail_to user.additional_email, user.additional_email %>
```

Comment:
This code looks like a logic (which should be in models) and not like a layout (which goes in views) thus it breaks MVC for some degree.
It is a bit slower (it needs to process more files instead of just calling a method in a known object), but not much actually.